modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -584,7 +585,6 @@ class ApiClient(object):
         :return: date.
         """
         try:
-            from dateutil.parser import parse
             return parse(string).date()
         except ImportError:
```

Comment:
This PR made the import mandatory. Should this `ImportError` handling get removed?